src/base/trafficGraph.js

```diff
@@ -371,6 +374,10 @@ class TrafficGraph extends EventEmitter {
   setState (state, force, parentState) {
     let updatedState = false;
     if (state && Object.keys(state).length > 0) {
+      // it's important to update entryNode when we're switching views
+      if (state.entryNode) {
+        this.entryNode = state.entryNode;
+      }
```

Comment:
Yes but I have to set up entryNode here when view changes. I dont mean all broken but ranknkig without this doesnt work because it undefined or equal to entryNode from previous graph